addons/circulation/stringtable.xml

```diff
@@ -1428,7 +1428,7 @@
         <Key ID="STR_kat_circulation_TRAINING_LEVEL_AED">
             <English>Training level required to use an AED</English>
             <Polish>Poziom wyszkolenia wymagany do korzystania z AED</Polish>
-            <German>Benötigtes Trainingslevel, um den AED zu nutzen</German>
+            <German>Benötigter medizinischer Grad, um den AED zu nutzen</German>
```

Comment:
IDK about changing that one, if it should be changed them pls in every other string table as well and not just randomly in this one.